source/includes/steps-ui-create.yaml

```diff
@@ -49,18 +49,18 @@ content: |
   - :guilabel:`Application Name`. 
     
     .. include:: /includes/note-app-name-limitations.rst
-  
+
   - Linked {+atlas-short+} cluster. If you do not have a cluster in this
     project, {+service+} creates one for you and automatically creates a
     linked data source with the name ``mongodb-atlas``. 
 
     .. include:: /includes/mongodb-4.4-required-for-sync-admonition.rst
-  
+
   - :ref:`Deployment model <realm-deployment-models>`. 
-  
+
   - :ref:`Deployment region <realm-regions>`.
 
-  Click :guilabel:`Create Realm Application`.
+  Click :guilabel:`Create {+app+}`.
```

Comment:
Will this button actually say "Create Atlas App Service"? We should probably run through all the UI procedures and make sure the UI elements we describe match the updated UI. Presumably as a fast follow/separate ticket?